operations/node_management/S3FS_Usage_and_Guidelines.md

```diff
@@ -0,0 +1,64 @@
+# S3FS Usage and Guidelines for Shasta
```

Comment:
We also need to create a new bullet point for this file in the operations/index.md#node_management section.